sqlglot/dialects/tsql.py

```diff
@@ -358,13 +370,75 @@ class Parser(parser.Parser):
         STATEMENT_PARSERS = {
             **parser.Parser.STATEMENT_PARSERS,
             TokenType.END: lambda self: self._parse_command(),
+            TokenType.ALIAS: lambda self: self._parse_alias(),
```

Comment:
why did you add this

Reply:
I put this in to try to work on CREATE PROCEDURE refinement. I'll remove for now.